tests/test_hfh_util.py

```diff
@@ -101,3 +101,67 @@ def test_copy_project_config_overwrite(copy, exists):
     assert copy.call_args == mock.call(
         "tests/huggingface-cache/dummy-fi.cfg", "projects.d/dummy-fi.cfg"
     )
+
+
+@mock.patch("annif.hfh_util._list_files_in_hf_hub", return_value=["README.md"])
+@mock.patch(
+    "huggingface_hub.ModelCard",
+)
+def test_upsert_modelcard_existing_card(ModelCard, _list_files_in_hf_hub, project):
+    repo_id = "annif-user/Annif-HFH-repo"
+    project.vocab_lang = "fi"
```

Comment:
The [project fixture](https://github.com/NatLibFi/Annif/blob/3b5f7a1ded9e841bc38a5c5e1a2c55a75cd0521c/tests/conftest.py#L137) does not provide `vocab_lang` so for these tests it is just set here, not super clean maybe?